rel-eng/build-packages-for-obs.sh

```diff
@@ -88,9 +88,22 @@ while read PKG_NAME PKG_VER PKG_DIR; do
 
   if [[ $PKG_DIR == *"containers"* ]]; then
     CONTAINER_NAME=$(basename "$PKG_DIR")
-    echo "=== Building container image [${CONTAINER_NAME}]"
-    if [ -d "$PKG_DIR" ]; then
-      cp -r "$PKG_DIR" "$SRPM_DIR/"
+    CONTAINS_SPEC_FILE=0
+
+    if ls $GIT_DIR/$PKG_DIR | grep \.spec 1> /dev/null 2>&1; then
+      CONTAINS_SPEC_FILE=1
+    fi
+    if [ ! -d "$GIT_DIR/$PKG_DIR" ]; then
+      FAILED_CNT=$(($FAILED_CNT+1))
+      FAILED_PKG="$FAILED_PKG$(echo -ne "\n    $(basename $PKG_DIR)")"
+      echo "*** FAILED Building package [$(basename $PKG_DIR)] - $PKG_DIR does not exist"
+      continue 2
+    elif [ $CONTAINS_SPEC_FILE == 0 ]; then
+      #check if folder does not contains a spec file, otherwise although it's in containers folder it's a RPM package
+      CONTAINER_NAME=$(basename "$PKG_DIR")
+      echo "=== Building container image [${CONTAINER_NAME}]"
+
+      cp -r "$GIT_DIR/$PKG_DIR" "$SRPM_DIR/"
```

Comment:
Why this change?

Reply:
These changes are already in Uyuni https://github.com/uyuni-project/uyuni/pull/6990 . Basically this commit https://github.com/uyuni-project/uyuni/commit/65901935cdcd5c3dd426950e46cd3af45f56f191 gaves the possibility to build container using  `build-packages-for-obs.sh` (instead of `build-containers-for-obs.sh` ) but it didn't work running with no argument. The PR fix the issue.